src/schema/nmdc.yaml

```diff
@@ -1008,12 +991,9 @@ classes:
       - growth_facil
       - humidity_regm
       - light_regm
-      #      - micro_biomass_meth
       - phosphate
       - samp_collec_method
       - samp_size
-      #      - soil_text_measure
-      #      - soil_texture_meth
```

Comment:
Why are these mixs slots being removed in this PR? 
Does 'comments' mean commented out lines, and not just LinkML comments?

Also not sure why we're reming them?